pkg/controller/controller_test.go

```diff
@@ -785,4 +785,51 @@ func TestDeleteFileShareSnapshot(t *testing.T) {
 	if _, err := ctrl.DeleteFileShareSnapshot(context.Background(), req); err != nil {
 		t.Errorf("failed to delete file share snapshot: %v\n", err)
 	}
+}
+
+func TestCreateFileShareAcl(t *testing.T) {
+	var req = &pb.CreateFileShareAclOpts{
+		Id: "6ad25d59-a160-45b2-8920-211be282e2df",
+		Description:      "This is a sample Acl for testing",
+		Type:             "ip",
+		AccessCapability: []string{"Read", "Write"},
+		AccessTo:         "10.32.109.15",
+		FileshareId:      "d2975ebe-d82c-430f-b28e-f373746a71ca",
+		Context:     c.NewAdminContext().ToJson(),
+	}
+	var fileshare = &SampleFileShares[0]
+	var acl = &SampleFileSharesAcl[0]
+	mockClient := new(dbtest.Client)
+	mockClient.On("GetFileShare", c.NewAdminContext(), req.FileshareId).Return(fileshare, nil)
+	mockClient.On("GetDockByPoolId", c.NewAdminContext(), fileshare.PoolId).Return(&SampleDocks[0], nil)
+	mockClient.On("UpdateStatus", c.NewAdminContext(), acl, "available").Return(nil)
```

Comment:
this looks incorrect...should we be mocking CreateFileShareAcl instead of UpdateStatus ?

Reply:
CreateFileShareAcl has been triggered at line  no.811 so after that update status is compulsory function to mock